src/ert/job_queue/job_queue_node.py

```diff
@@ -394,3 +473,22 @@ def wait_for(self) -> None:
 
     def _set_thread_status(self, new_status: ThreadStatus) -> None:
         self._thread_status = new_status
+
+
+class ProcessWithException(mp.Process):
```

Comment:
It would be handy to have a small doc / comment for why we have this class in the first place.

Reply:
I could not add more information then

- it's for doing stuff in a subprocess, and
- we add exception handling

so from that point of view i think the name does the trick

but maybe you would like to see some other information, that i'm missing / not thinking of? what would you like to include / see in such a doc or comment?

Reply:
What I meant is the by default we won't get any stacktrace from a Process, therefore `wait_and_throw_if_exception` needs to be called explicitly or something along those lines :)

Reply:
ah, indeed - the usage. good point, thank you!

added in 6b543a112aa94b79b275312fa0770ab58660651c